### ISSUE TITLE
Anzeigen für Seelenkraft, Zähigkeit, Ausweichen, Initiative, WS

### ISSUE THREAD

**Issue by CommanderAlex98** [phrogg/DSAAssistant#10]

**Comment by phrogg** [comment]
Danke, ich werde demnächst eine neue einzelne Version extra für DSA 5 aufsetzen :)

**Comment by Rathch** [comment]
Ist die neue dsa5 Version schon spruchreif

**Comment by phrogg** [comment]
Hatte bis jetzt leider viel um die Ohren und bin noch nicht viel weiter gekommen. Aber, wenn die Klausurenphase nächsten Monat vorbei ist. Nehme ich das ganze voll und ganz in den Angriff.

**Comment by Naxos84** [comment]
Gibt es hier schon Neuigkeiten? (Nach 3 Jahren)

**Comment by phrogg** [comment]
> Gibt es hier schon Neuigkeiten? (Nach 3 Jahren)

Gude, ich habe mal eine ganze Zeit dran gearbeitet, dadurch das ich allerdings selbst kein DSA 5 spiele nicht wirklich dran hängen geblieben, ich hatte damals ein Repo aufgemacht aber noch nichts gepusht: https://github.com/phrogg/DSA5Assistant jemand Interesse daran mitzuarbeiten? Dann wäre ich auch motivierter, ich kenne so die ganze Materie auch nicht wirklich. Und dann ich auch nie Spenden oder ähnliches erhalten habe, bin ich davon ausgegangen das die Nachfrage bei der App eher gering ist ...

**Comment by phrogg** [comment]
Vielleicht könnte OP (@CommanderAlex98) auch mal ausführen, was damit gemeint ist, und wo die Indikatoren eingefügt werden sollten.

**Comment by Naxos84** [comment]
Also kann man davon ausgehen, daß das Projekt nicht mehr weiter entwickelt wird? Oder habe ich das falsch verstanden?

**Comment by phrogg** [comment]
> Also kann man davon ausgehen, daß das Projekt nicht mehr weiter entwickelt wird? Oder habe ich das falsch verstanden?

Nein, das meine ich nicht. Die App wird definitiv weiterentwickelt. Und je nachdem, wie viel Aufwand das ist, passiert das Ganze früher oder später. Im Moment sehe ich für mich keine Motivation daran weiterzuarbeiten, da ich die App wie gesagt selbst nicht in der Form brauche und auch monetäre Motivation nicht vorhanden ist. Wenn mir jemand eine Liste zusammenstellen kann, welche Änderungen in welchem Umfang nötig sind und ich das abschätzen kann. Ich kann auch etwas Zeit neben Arbeit und Studium einrichten...
